docs/.style/variables.sass

```diff
@@ -5,9 +5,13 @@ $drop-shadow: rgba(22, 22, 22, 0.2)
 $white: #FFF
 $pearl: #f6f7f7
 $menu-open: #347193
+$green: #92DC84
```

Comment:
I think you can overrider success, danger and warning from bootstrap:
https://github.com/fsprojects/fantomas/blob/e2eda798e15b9bf36fc2bbfc809c87fe23681fcc/docs/.style/bootstrap/scss/_variables.scss#L303-L306